scrub_hub_project/authenticate/views.py

```diff
@@ -0,0 +1,67 @@
+from django.shortcuts import render
+
+# Create your views here.
+import json
+from django.contrib.auth import authenticate, login, logout
+from django.http import JsonResponse
+from django.views.decorators.csrf import ensure_csrf_cookie
+from django.views.decorators.http import require_POST
+from django.contrib.auth.models import User
+# Create your views here.
+
+def login_view(request):
+    if request.method == "POST":
```

Comment:
```suggestion

@ensure_csrf_cookie
def login_view(request):
    if request.method == "POST":
```
Make sure the user will have a CSRF token when they visit a page from which a POST request will be made.